G1ANT.Addon.Watson/G1ANT.Addon.Watson/Commands/WatsonSpeechToTextCommand.cs

```diff
@@ -20,29 +22,26 @@ public class Arguments : CommandArguments
             public TextStructure Path { get; set; }
 
             [Argument(Required = true, Tooltip = "Specifies service's login.")]
-            public TextStructure Login { get; set; }
+            public TextStructure ApiKey { get; set; }
```

Comment:
Changing attribute names for commands can create some regression issues.

Reply:
This command didn't work for years. I don't think we need to worry about regression.

Reply:
Seams legit 👍